### ISSUE TITLE
Porting to postmaketOS

### ISSUE THREAD

**Issue by ticotaco72** [SunnyRaj84348/android_kernel_xiaomi_msm8917#9]
Hello, I'm trying to port xiaomi-riva to pmOS and I found your kernel. I wanted to ask few questions.

Is 4.9 kernel at https://github.com/SunnyRaj84348/android_kernel_xiaomi_msm8917/tree/lineage-17.1_4.9 ready for use?

Is [this](https://github.com/SunnyRaj84348/android_kernel_xiaomi_msm8917/blob/lineage-17.1/arch/arm64/configs/riva_defconfig) the defconfig used?

Is firmware included in Kernel? Or should it be already on device or manually appended by distribution?

**Comment by SunnyRaj84348** [comment]
> Hello, I'm trying to port xiaomi-riva to pmOS and I found your kernel. I wanted to ask few questions.
> 
> Is 4.9 kernel at https://github.com/SunnyRaj84348/android_kernel_xiaomi_msm8917/tree/lineage-17.1_4.9 ready for use?
> 
> Is [this](https://github.com/SunnyRaj84348/android_kernel_xiaomi_msm8917/blob/lineage-17.1/arch/arm64/configs/riva_defconfig) the defconfig used?
> 
> Is firmware included in Kernel? Or should it be already on device or manually appended by distribution?

Yeah, that's the defconfig and firmware is default as provided by OEM (nougat firmware)
